examples/accounts/create-sub-account/Create sub account.md

```diff
@@ -4,10 +4,46 @@ This example demonstrates how to create a sub account under your Twilio account
 
 ## Prerequisites
 
-### 1. Set up
-Refer to the setup guide in [ReadMe](../../../README.md) for necessary credentials.
+Before using the ballerinax-twilio connector you must have access to Twilio API, If you do not have access to Twilio API please complete the following steps:
 
-### 2. Configuration
+### Step 1: Create a Twilio account
+
+Creating a Twilio account can be done by visiting [Twilio](https://www.twilio.com) and clicking the "Try Twilio for Free" button.
+
+### Step 2: Obtain a Twilio phone number
+
+All trial projects can provision a free trial phone number for testing. Here's how to get started.
+
+> **Notice:** Trial project phone number selection may be limited. You must upgrade your Twilio project to provision more than one phone number, or to provision a number that is not available to trial projects.
+
+1. Access the Buy a Number page in the Console.
+
+![Get Phone Number](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-twilio/master/ballerina/resources/get-phone-number.png)
+
+2. Enter the criteria for the phone number you need, and then click Search.
+
+![Configure Phone Number](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-twilio/master/ballerina/resources/phone-number-config.png)
+
+- Country: Select the desired country from the drop-down menu.
+- Number or Location: Select the desired option to search by digits/phrases, or a specific City or Region.
+- Capabilities: Select your service needs for this number. 
+
+3. Click Buy to purchase a phone number for your current project or sub-account.
+
+![Search Results](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-twilio/master/ballerina/resources/search-phone-number.png)
+> **Notice:** Many countries require identity documentation for Phone Number compliance. Requests to provision phone numbers with these regulations will be required to select or add the required documentation after clicking Buy in Console. To see which countries and phone number types are affected by these requirements, please see twilio's [Phone Number Regulations](https://www.twilio.com/guidelines/regulatory) site.
+
+### Step 3: Obtain a Twilio account Sid with auth token
+
+Twilio uses two credentials to determine which account an API request is coming from: The account Sid, which acts as a `username`, and the Auth Token which acts as a `password`. You can find your account Sid and auth token in your [Twilio console](https://www.twilio.com/console).
```

Comment:
```
The account Sid, which acts as a `username`, and the Auth Token which acts as a `password`.
```

Since we are moving with their naming convention for our credentials we don't need this.

Reply:
Resolved!